pandas/core/groupby/ops.py

```diff
@@ -214,6 +215,10 @@ def apply(self, f: F, data: FrameOrSeries, axis: int = 0):
             #  TODO: can we have a workaround for EAs backed by ndarray?
             pass
 
+        elif isinstance(sdata._mgr, ArrayManager):
+            # TODO(ArrayManager) don't use fast_apply / libreduction.apply_frame_axis0
+            # for now -> relies on BlockManager internals
```

Comment:
im totally fine with skipping for the time being.  medium-term, i think it could use .arrays instead of .blocks, might be easy-ish compat